content/shmem_sync_all.tex

```diff
@@ -0,0 +1,44 @@
+\color{ForestGreen}
+\apisummary{
+    Registers the arrival of a \ac{PE} at a barrier and suspends \ac{PE}
```

Comment:
This is the same text that we have for barrier.  Because the proposals are happening in parallel, it's a little tricky.  If sync moves ahead at this meeting, we can handle it as part of the threading proposal before voting.  Otherwise, we could use the section committee.

I think you mean, suspend the calling thread rather than all threads in the PE.  That's the semantic that I would expect to see for barriers in the threading proposal.